### PR TITLE
[AKS] Support updating tags by az aks update --tags

### PR DESCRIPTION
**Pull request by nilo19**
**Description**

Support updating tags of a managed cluster by `az aks update --tags`

**Testing Guide**

Make sure `az aks update --tags` could update the tags of an existing managed cluster.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -378,6 +378,9 @@
   - name: --aci-subnet-name
     type: string
     short-summary: The name of a subnet in an existing VNet into which to deploy the virtual nodes.
+  - name: --tags
+    type: string
+    short-summary: The tags of the managed cluster. The managed cluster instance and all resoruces managed by the cloud provider will be tagged.
```
Comment (matthew-rollitt) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:383]: Resources

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 2]
```diff
@@ -498,6 +503,9 @@
   - name: --disable-ahub
     type: bool
     short-summary: Disable Azure Hybrid User Benefits (AHUB) feature for cluster.
+  - name: --tags
+    type: string
+    short-summary: The tags of the managed cluster. The managed cluster instance and all resoruces managed by the cloud provider will be tagged.
```
Comment (matthew-rollitt) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:508]: Resources

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 3]
```diff
@@ -430,6 +433,8 @@
     text: az aks create -g MyResourceGroup -n MyManagedCluster --node-osdisk-diskencryptionset-id <disk-encryption-set-resource-id>
   - name: Create a kubernetes cluster with ephemeral OS enabled.
     text: az aks create -g MyResourceGroup -n MyManagedCluster --node-osdisk-type Ephemeral --node-osdisk-size 48
+  - name: Create a kubernetes cluster with custom tags
+    text: az aks create -g MyResourceGroup -n MyManagedCluster --tags "a=b" "c=d"
```
Comment (matthew-rollitt) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:437]: maybe foo=bar or examples of actual tags would be good

Comment (nilo19) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py]: fixed, thanks

**Review by qwordy — APPROVED** [review]

**Comment by FumingZhang** [comment]
Hi @nilo19, could you please help resolve the conflicts?

**Pull request closed without merging**